cpp/daal/src/externals/service_rng_mkl.h

```diff
@@ -32,13 +32,19 @@
 #define __DAAL_BRNG_MT2203                    VSL_BRNG_MT2203
 #define __DAAL_BRNG_MT19937                   VSL_BRNG_MT19937
 #define __DAAL_BRNG_MCG59                     VSL_BRNG_MCG59
+#define __DAAL_BRNG_MRG32K3A                  VSL_BRNG_MRG32K3A
+#define __DAAL_BRNG_PHILOX4X32X10             VSL_BRNG_PHILOX4X32X10
 #define __DAAL_RNG_METHOD_UNIFORM_STD         VSL_RNG_METHOD_UNIFORM_STD
 #define __DAAL_RNG_METHOD_UNIFORMBITS32_STD   0
 #define __DAAL_RNG_METHOD_BERNOULLI_ICDF      VSL_RNG_METHOD_BERNOULLI_ICDF
 #define __DAAL_RNG_METHOD_GAUSSIAN_BOXMULLER  VSL_RNG_METHOD_GAUSSIAN_BOXMULLER
 #define __DAAL_RNG_METHOD_GAUSSIAN_BOXMULLER2 VSL_RNG_METHOD_GAUSSIAN_BOXMULLER2
 #define __DAAL_RNG_METHOD_GAUSSIAN_ICDF       VSL_RNG_METHOD_GAUSSIAN_ICDF
 
+// Errors
+#define __DAAL_LEAPFROG_METHOD_ERRCODE   -1002
+#define __DAAL_SKIP_AHEAD_METHOD_ERRCODE -1003
```

Comment:
Please use MKL defines:

```suggestion
#define __DAAL_LEAPFROG_METHOD_ERRCODE   VSL_RNG_ERROR_LEAPFROG_UNSUPPORTED
#define __DAAL_SKIP_AHEAD_METHOD_ERRCODE VSL_RNG_ERROR_SKIPAHEAD_UNSUPPORTED
```